cmd/installer/cli/install.go

```diff
@@ -97,6 +98,9 @@ type InstallCmdFlags struct {
 	tlsKeyBytes  []byte
 }
 
+// webAssetsFS is the filesystem to be used by the web component. Defaults to nil allowing the web server to use the default assets embedded in the binary. Useful for testing.
+var webAssetsFS fs.FS = nil
```

Comment:
Not particularly happy about this, unsure if we should keep this somewhere else, open to ideas.